README.md

```diff
@@ -341,6 +341,7 @@ try {
 
 Changelog
```

Comment:
This is a general comment. I think that we should create 'CHANGELOG.md' file under each our SDK repository and drop this section here.